src/Iot.Device.Bindings/Utils/Color.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Runtime.InteropServices;
```

Comment:
System.Drawing already defines Color class

Reply:
I know :$, I just don't like it ;). Anyway, I ported lib and sample to use System.Drawing. I loose a bunch of helpers. But maybe we can get those in System.Drawing later.

Reply:
@maloo not a fan of Color and and Vector/Vector2/Vector3 as well - see my sample in SenseHat PR - the code could be at least 3 times shorter if we could do something similar to GLSL. You should create an issue on https://github.com/dotnet/corefx